code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2919,36 +2898,16 @@
 	color = "#B31008" //rgb: 139, 166, 233
 	data = 1 //Used as a tally
 	custom_metabolism = FOOD_METABOLISM
+	temperature = T0C+10
```

Comment:
If we're going to take the approach that cold peppers merely produce a cold sensation rather than actually having a cold temperature, I might suggest adding some kind of descriptive text to them mentioning that they contain menthol, the compound found in mints that produces a cool sensation.